ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```
@@ -506,6 +506,57 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-upgrade-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

```suggestion
    always_run: false
```
This has to be manually set after `make update` or `make jobs` if you do not want the job to run on all PRs. I would expect that at least initially you only want this on demand i.e. via `/test gke-version-compatibility-tests`.

---

> I would expect that at least initially you only want this on demand i.e. via /test gke-version-compatibility-tests.

Exactly, thanks!